src/families/bitcoin/wallet-btc/storage/index.ts

```diff
@@ -17,22 +17,39 @@ class BitcoinLikeStorage implements IStorage {
   // returning unordered tx within the same block)
   spentUtxos: { [key: string]: Input[] } = {};
 
-  async getLastTx(txFilter: {
-    account?: number;
-    index?: number;
-    address?: string;
-    confirmed?: boolean;
-  }) {
+  async getLastUnconfirmedTx(txFilter: { account?: number; index?: number }) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    const tx: TX | undefined = findLast(this.txs, (t) => {
+      return (
+        (!txFilter.account || t.account === txFilter.account) &&
+        (!txFilter.index || t.index === txFilter.index) &&
```

Comment:
index can be `0` , but you should be able to just do `=== undefined`

Reply:
Ouch... pre-holiday version of the late friday commit. :trollface: 
Thanks!